sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageStreamSource.java

```diff
@@ -153,6 +156,17 @@ public String toString() {
   @Experimental(Kind.SOURCE_SINK)
   public static class BigQueryStorageStreamReader<T> extends BoundedSource.BoundedReader<T> {
 
+    private final Counter streamingResponseCount =
+        Metrics.counter(
+            BigQueryStorageStreamReader.class, "bq_storage_read_streaming_response_count");
+    private final Counter streamingResponseTotalLatencyMs =
+        Metrics.counter(
```

Comment:
maybe [distribution](https://beam.apache.org/releases/javadoc/2.0.0/org/apache/beam/sdk/metrics/Distribution.html) for time base metrics?

Reply:
Done.